integrations/puppet-module/sysdig-falco/manifests/install.pp

```diff
@@ -1,6 +1,61 @@
 # == Class: falco::install
 class falco::install inherits falco {
+  case $::osfamily {
+    'Debian': {
+      apt::source { 'sysdig':
+        location          => 'http://download.draios.com/stable/deb',
+        release           => 'stable-$(ARCH)/',
+        repos             => '',
+        required_packages => 'debian-keyring debian-archive-keyring',
```

Comment:
I installed using puppet 4, which in turn uses puppetlabs-apt v6.3.0. In that version, `required_packages`, `key_source`, and `include_src` are all deprecated and attempts to use them results in an error:

```
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Error while evaluating a Resource Statement, Apt::Source[sysdig]:
  has no parameter named 'required_packages'
  has no parameter named 'key_source'
  has no parameter named 'include_src' at /etc/puppetlabs/code/environments/production/modules/falco/manifests/install.pp:5 on node ubuntu
```

How about we remove them and just state that puppet >= 4 is required?

Reply:
Okay! So, I tried to stay compatible with Puppet 3. The latest version of `apt` that is compatible with Puppet 3 is 2.4.0, but it turns out there is a bug in that version of `apt`: https://tickets.puppetlabs.com/browse/MODULES-4913

So, do we upgrade to Puppet 4? On my side, I'll override apt to make it work, but here, we can break the compatibility with Puppet 3 if you want 🙂

Reply:
Okay, done! The patched version of apt is 4.1.0, I upgraded to 4.5.1, which is compatible with Puppet 4.7.0. I did test 4.5.1, it works, and I'm kind of running out of time to test 6.3.0 😄 If you have the time, please do, otherwise, I think we can merge it as is 🙂